### DISCUSSION TITLE
Bring back Prism Component from v6

### DISCUSSION THREAD

**Discussion by nicklpeterson** [mantinedev/mantine#5334] [Feature requests]
Thanks for all your great work on this project 😄 !

Unfortunately the new highlight.js component is blocking me from upgrading an App to to v7. My app uses line highlighting and line numbers to show diffs between file versions. It doesn't seem like there is a good way to do either of those with the new component, also personally I don't think the highlighting is quite as nice out. 

Would it be possible to port @mantine/prism to v7, then people would be able to choose which component to use for code highlighting? I'd be happy to contribute to the effort if time and resources are the issue.

**Comment by rtivital** [comment]
You are welcome to publish a package on npm that implements this migration. It is not plan to maintain two almost identical packages in the Mantine repository.

**Reply by nicklpeterson** [reply]
That's fair, thanks again for such a great library.
